synapse/lib/sched.py

```diff
@@ -100,15 +100,30 @@ def woot(x,y):
 
     def persec(self, count, func, *args, **kwargs):
         '''
-        Scedule a callback to occur count times per second.
+        Schedule a callback to occur count times per second.
 
-        Example:
+        Args:
+            count: Number of times per second for this to occur. Either an int or a float.
+            func: Function to execute.
+            *args: Args passed to the function.
+            **kwargs: Kwargs passed to the function.
 
-            def tenpersec(x,y=None):
-                blah()
+        Examples:
+            Scheduled a function to be called 10 times per second::
 
-            sched = Sched()
-            sched.persec(10, tenpersec, 10, y='woot')
+                def tenpersec(x,y=None):
+                    blah()
+
+                sched = Sched()
+                sched.persec(10, tenpersec, 10, y='woot')
+
+        Notes:
+            This indefinitely calls the scheduled function until the function
+            returns None or the Task is fini'd. See the Sched.loop function
```

Comment:
should be until the function returns False ( any function falling through and returning None is probably unaware of the loop breaking behavior of it's return value )

Reply:
opps - fixing